src/components/__tests__/Tooltip.test.tsx

```diff
@@ -188,22 +201,31 @@ describe('Tooltip', () => {
             const currentProps = tooltip.props
             const shouldUpdate = tooltip.shouldComponentUpdate(
                 currentProps,
-                currentState
+                currentState,
+                undefined
```

Comment:
What and why is this third argument? shouldComponentUpdate only takes two (nextProps, nextState).

Reply:
The types somehow seem to say it accepts 3:

```
shouldComponentUpdate?(nextProps: Readonly<P>, nextState: Readonly<S>, nextContext: any): boolean;
```

Reply:
Something's clearly off because we only have 2 in Tooltip.tsx. But I guess we can live with this.

Alternative could be `shallow(getTooltip()).instance() as Tooltip` but I'm not sure it's better.